app/src/main/AndroidManifest.xml

```diff
@@ -1,24 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.simplecity.amp_library">
-
     <!-- ## 'Protection-Normal' permissions ## -->
```

Comment:
Looks like a lot of unrelated (formatting?) changes here - could you please remove them?